docs/streams/developer-guide/config-streams.html

```diff
@@ -680,14 +680,18 @@ <h4><a class="toc-backref" href="#id33">default.windowed.value.serde.inner</a><a
               preferred to restart on an instance that has standby replicas so that the local state store restoration process from its
               changelog can be minimized.  Details about how Kafka Streams makes use of the standby replicas to minimize the cost of
               resuming tasks on failover can be found in the <a class="reference internal" href="../architecture.html#streams_architecture_state"><span class="std std-ref">State</span></a> section.
+              <dl class="docutils">
+                <dt>Recommendation:</dt>
+                <dd>Increase the number of standbys to 1 to get instant fail-over, ie, high-availability.
```

Comment:
@showuon did you know you can directly make suggestions? much faster and more convenient than writing it out how you did.
```suggestion
                <dd>Increase the number of standbys to 1 to get instant fail-over, i.e., high-availability.
```